### DISCUSSION TITLE
Project Update

### DISCUSSION THREAD

**Discussion by danielgtaylor** [danielgtaylor/huma#494] [Announcements]
Hi folks,

I just wanted to leave a note as there hasn't been a ton of activity on the project in the last few weeks. I don't want to share too much personal information publicly on the Internet but do want Huma users to know what is going on. I'm in the middle of a big move to a new house that has taken most of my time which is now just about done. We are mostly unpacked and things can start to get back to normal, so I expect to get to the issues, PRs, and discussions soon.

Schools are also out for the summer so there will be times I'm on vacation and traveling, so expect slower responses sometimes! I'd appreciate if people can help with issue triage and code reviews.

Thanks for the patience and understanding!

**Comment by pawel-biegun-sq3** [comment]
Your project is awesome, it brings an enormous productivity boost compared to raw chi/another router.

**Comment by joa23** [comment]
Happy Summer - well deserved.
